README.md

```diff
@@ -152,7 +152,11 @@ brew services stop redis
 
 **NOTE:** Lisk does not run on the redis default port of 6379. Instead it is configured to run on port: 6380. Because of this, in order for Lisk to run, you have one of two options:
 
-**Change the Redis launch configuration**
+1. **Change the Lisk configuration**
+
+Update the redis port configuration in the `config.json` file. Note that this is the easiest option, however, be mindful of reverting the changes should you make a pull request.
```

Comment:
Unclear and misleading.

Reply:
This is not part of the current PR.

Reply:
So why that is in changes?

Reply:
They come after solving some conflicts when I've merged the latest changes.

Reply:
So looks like there is something wrong...